lib/utils/extensions.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:pokedex/utils/strings.dart' as str;
+import 'package:pokedex/utils/constant.dart' as k;
+
+extension ModifyUrl on String {
```

Comment:
this will only limit to url stuff, what about for other strings manipulation? How about rename this to `StringManipulation`? 🤔